### PR TITLE
Add Kotlin primitive types to parser

### PR DESCRIPTION
**Pull request by shans**

### CODE REVIEW

File: src/runtime/manifest-parser.pegjs [hunk 1]
```diff
@@ -1545,6 +1546,16 @@ SchemaPrimitiveType
     });
   }
 
+KotlinPrimitiveType
```
Comment (cromwellian) [inline src/runtime/manifest-parser.pegjs:1549]: Wouldn't it be better to call these Integral types? They're not really specific to Kotlin, and exist in most other languages, Java, C, protos, even in JS/TS/WASM if you count TypedArray buffers.

Comment (shans) [reply src/runtime/manifest-parser.pegjs:1549]: No, that's counter to the approach we want to take here (see https://docs.google.com/document/d/1FkglKohOwYT7pU4pJE3AxIiB7cAcssNQkWm8ISGvVgU/edit#heading=h.loqnco9pcpet).

To reiterate: these are the Kotlin types, and they'll be guarded by a Kotlin namespace eventually. They aren't, in general, the same set as the types in other languages. For example, Kotlin doesn't have unsigned types but C/C++ does. Ultimately each target language will have its own namespaced set of native types and we'll use our refinement system to understand / mediate transformations between them.

Comment (cromwellian) [reply src/runtime/manifest-parser.pegjs:1560]: Just a nit, but Kotlin does have unsigned types, they were introduced in Kotlin 1.3. (UShort, UByte, UInt, etc), like coroutines, marked experimental, but it's everyone's expectations they're here to stay. 

https://kotlinlang.org/docs/reference/whatsnew13.html

Comment (shans) [reply src/runtime/manifest-parser.pegjs:1560]: good to know. There are still big differences across the languages :)

**Pull request merged**